ludwig/data/parquet_dataset.py

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env python
+# coding=utf-8
+# Copyright (c) 2019 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import pyarrow.parquet as pq
+
+
+class ParquetDataset:
+    def __init__(self, dataset, input_features, output_features, data_parquet_fp):
```

Comment:
Nit: what is the data type of the dataset here, can we add something to the name to make it more clear?